libr/include/r_util/r_assert.h

```diff
@@ -0,0 +1,69 @@
+#ifndef R_ASSERT_H
```

Comment:
shoudl there be a "r_goto_if_fail" or equivalent? there's a lot of goto beach; that can use this

Reply:
I don't think so. This function should be used to represent preconditions and "programming error" as described in https://developer.gnome.org/glib/stable/glib-Warnings-and-Assertions.html#g-return-if-fail . It shouldn't be used for generic error handling (e.g. malloc return, etc.)